packages/grid/x-grid-modules/src/components/styled-wrappers/GridRootStyles.ts

```diff
@@ -174,8 +176,7 @@ export const useStyles = makeStyles(
         },
         '& .MuiDataGrid-cellWithRenderer': {
           display: 'flex',
-          flexDirection: 'column',
-          justifyContent: 'center',
+          alignItems: 'center',
```

Comment:
This change might be controversial. I was surprised by the previous behavior. I couldn't understand why my custom cell was full width;